conf/zapi/default.yaml

```diff
@@ -21,3 +22,15 @@ objects:
   Status_7mode:        status_7.yaml
   Subsystem:           subsystem.yaml
   Volume:              volume.yaml
+
+
+  # for security dashboard at cluster
```

Comment:
are these comments still needed?
we should decide if the security dashboard templates should be set off by themselves or alphabetized with the others. If set off, shouldn't the `svm.yaml` key/value be moved down into the security section?

If not set-off, mix and alphabetize list

Reply:
I am fine with both the cases.

If set off, shouldn't the svm.yaml key/value be moved down into the security section?
--> As svm is object and fields from svm are not only scope to security dashboard. So even if we all agreed to set-off, svm should be near object list.

Reply:
sorted in alphabetic order for zapi/rest